packages/types/src/transaction/classify.ts

```diff
@@ -7,6 +7,9 @@ export const classifyTransaction = (txv?: TransactionView): TransactionClassific
     return 'unknown';
   }
 
+  if (txv.bodyView?.actionViews.some(a => a.actionView.case === 'swap')) return 'swap';
+  if (txv.bodyView?.actionViews.some(a => a.actionView.case === 'swapClaim')) return 'swapClaim';
```

Comment:
(Is it ever possible for a single transaction to have multiple types of actions at once? e.g., where you send money to another person, delegate some tokens to a validator, and do a swap — all in the same transaction? if so, this `classifyTransaction` function is pretty useless, since it's assuming there's only one primary purpose per transaction.)